src/views/TestDate/index.tsx

```diff
@@ -0,0 +1,333 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link, useHistory, useParams } from 'react-router-dom';
+import { useMutation, useQuery } from '@apollo/client';
+import { Button } from '@trussworks/react-uswds';
+import { Field, Form, Formik, FormikProps } from 'formik';
+import { DateTime } from 'luxon';
+import CreateTestDateQuery from 'queries/CreateTestDateQuery';
+import GetAccessibilityRequestQuery from 'queries/GetAccessibilityRequestQuery';
+import { GetAccessibilityRequest } from 'queries/types/GetAccessibilityRequest';
+
+import Footer from 'components/Footer';
+import Header from 'components/Header';
+import MainContent from 'components/MainContent';
+import PageWrapper from 'components/PageWrapper';
+import { ErrorAlert, ErrorAlertMessage } from 'components/shared/ErrorAlert';
+import FieldErrorMsg from 'components/shared/FieldErrorMsg';
+import FieldGroup from 'components/shared/FieldGroup';
+import HelpText from 'components/shared/HelpText';
+import Label from 'components/shared/Label';
+import { RadioField } from 'components/shared/RadioField';
+import { NavLink, SecondaryNav } from 'components/shared/SecondaryNav';
+import TextField from 'components/shared/TextField';
+import { TestDateForm } from 'types/accessibilityRequest';
+import flattenErrors from 'utils/flattenErrors';
+import { TestDateValidationSchema } from 'validations/testDateSchema';
+
+import './index.scss';
+
+const TestDate = () => {
+  const { t } = useTranslation('accessibility');
+  const { accessibilityRequestId } = useParams<{
+    accessibilityRequestId: string;
+  }>();
+  const [mutate, mutationResult] = useMutation(CreateTestDateQuery, {
```

Comment:
I think we want to specify the type when calling the mutation:

```suggestion
  const [mutate, mutationResult] = useMutation<CreateTestDate>(CreateTestDateQuery, {
```

That should help typescript detect more thoroughly.